core/src/main/java/bio/terra/pearl/core/model/export/ExportIntegration.java

```diff
@@ -0,0 +1,21 @@
+package bio.terra.pearl.core.model.export;
+
+import bio.terra.pearl.core.model.BaseEntity;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+
+import java.util.UUID;
+
+@Getter
+@Setter
+@SuperBuilder
+@NoArgsConstructor
+public class ExportIntegration extends BaseEntity {
+    private UUID studyEnvironmentId;
+    private boolean enabled = true;
+    private ExportDestinationType destinationType;
+    private String destinationProps;
+    private String exportOptions;
```

Comment:
would we want to use export options in other places later? I almost kinda feel like it might be worth making a full db object, and we could do other fun stuff (save user's preferred export options? or, users can create export presets for different aspects of their workflow, e.g. "Kit Export" or something). The simplicity is nice, but it's got me thinking all the cool stuff we can do 😄 .

Reply:
yeah, good ideas. I'll keep it as a string for now, and then we can make it a full table once the export options crystallize a bit more (we've still got to add support for including/excluding individual survey questions)